invokeai/frontend/web/src/services/api/endpoints/images.ts

```diff
@@ -113,11 +114,33 @@ export const imagesApi = api.injectEndpoints({
       ],
       keepUnusedDataFor: 86400, // 24 hours
     }),
-    getImageMetadata: build.query<UnsafeImageMetadata, string>({
+    getImageMetadata: build.query<ImageMetadataAndWorkflow, string>({
```

Comment:
Will OSS still be pulling the image and parsing it from the metadata on the front-end? Wondering if we can get rid of that logic if this endpoint is going to be used instead

Reply:
No more reading from the image itself. Will clean up.